raven/processes/wps_zonal_stats.py

```diff
@@ -0,0 +1,115 @@
+import logging
+
+from rasterstats import zonal_stats
+from pywps import LiteralInput, ComplexInput
+from pywps import LiteralOutput
+from pywps import Process
+from pywps.app.Common import Metadata
+
+LOGGER = logging.getLogger("PYWPS")
+
+
+class ZonalStatistics(Process):
+    """Given a file containing vector data, provide general information and spatial characteristics"""
+
+    def __init__(self):
+        inputs = [
+            LiteralInput('touches', 'Select boundary pixels that are touched by shape', data_type='boolean'),
+            LiteralInput('categorical', 'Return distinct pixel categories', data_type='boolean'),
+            LiteralInput('band', 'Raster band', data_type='int', default=1,
+                         abstract='Band of raster examined to perform zonal statistics. Defaults to 1'),
```

Comment:
zonal statistics ? Does it mean what I think it means ? https://en.wikipedia.org/wiki/Zonal_and_meridional

Reply:
No. In GIS circles, zonal statistics are typically the term used for gathering statistics for a delimited region. Thinking about it now, it is a bit odd since zonal in this sense can also mean mercator-like slices. Good question.